src/webots/launcher/webots-linux.sh

```diff
@@ -99,8 +91,4 @@ wait ${webots_pid}
 
 webots_return_code=$?
 
-# clean-up tmp folder and pipe files in case webots crashed without clean-up
-rm -rf $WEBOTS_TMP_PATH
-rm -f ${TMPDIR}/webots_${webots_pid}_*
```

Comment:
Why is it not needed anymore?

Reply:
Because the temporary folder is not any more based on the PID and the script doesn't know which port number Webots uses. Moreover this clean-up was not strictly needed since Webots does the clean-up pretty well on quit and on start-up.